RadarSDK/Include/RadarTripOptions.h

```diff
@@ -52,6 +52,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nullable, nonatomic, copy) NSDate *scheduledArrivalAt;
 
+/**
+ Optional radius in meters at which to begin ramping up tracking. Takes precedence over both geofence specific and tracking option ramp-up radii.
+ */
+@property (nonatomic, assign) int rampUpRadius;
```

Comment:
should this also have `rampUpInterval`?

Reply:
No, we don't want a `rampUpInterval` here actually. We check for the trip option `rampUpRadius` in our `replaceSyncedGeofences` _iff_ the geofence in question is the trip destination geofence.

The `rampUpInterval` comes up in `handleLocation`, and applies regardless of which geofence has caused the ramp up.

If one wanted to have a different `rampUpInterval` when on trips, they could achieve that via passing in tracking options with the desired `rampUpInterval`.